src/actions/plugins/sneak.ts

```diff
@@ -0,0 +1,105 @@
+import * as vscode from 'vscode';
+import { VimState } from '../../state/vimState';
+import { Configuration } from './../../configuration/configuration';
+import { ModeName } from './../../mode/mode';
+import { RegisterAction } from './../base';
+import { BaseCommand } from './../commands/actions';
+import { Position } from '../../common/motion/position';
+import { IMovement, BaseMovement, createRepeatMovement } from '../motion';
+
+@RegisterAction
+class SneakForward extends BaseMovement {
+  keys = [['s', '<character>', '<character>'], ['z', '<character>', '<character>']];
```

Comment:
Will `['s', '<character>', '<character>']` not conflict with surround?

Reply:
No, because that only matches when there is no operator (see `couldActionApply`). With an operator it uses `z` instead. Surround uses `ys` without an operator and `s` with an operator.

Reply:
I'm not familiar with the `sneak` plugin. Can we use only `z`? Might be confusing for folks that it is sometimes `s` and other times `z`.

Reply:
No, we can't use `z` because that's used for folds. I think the main audience for this plugin are people who have used `vim-sneak` and they'll be used to it. I think it would be more confusing for people if the Sneak plugin didn't act like `vim-sneak`.

Admittedly it's not a perfect design, but I can't think of a better one (at least without changing a bunch of other vim keybindings). It's always tricky to find unused keys in vim. Surround's `ys` vs `<operator>s` vs `S` is also confusing, but you can get used to it.